Week3/moreJs-6.js

```diff
@@ -0,0 +1,19 @@
+/*
+Create a function called vehicle, like before,
+ but takes another parameter called age, 
+ so that vehicle("blue", 1, 5) 
+ prints 'a blue used car'
+*/
+
+function vehicle (color, type, age){
+    let types = [' car' , ' motorbike'];
+    console.log('a ' + color +' ' + condition(age) + types[(type-1)]);
```

Comment:
What happens if you pass '3' as value for the `type` as arguments? Will this still work?